pages/_app.jsx

```diff
@@ -28,11 +36,19 @@ const DynamicWalletContextProvider = dynamic(
 function MyApp ({ Component, pageProps }) {
   return (
     <DynamicWalletContextProvider>
-      <Provider>
-        <Layout>
-          <Component {...pageProps} />
-        </Layout>
-      </Provider>
+      <ReactQueryCacheProvider queryCache={queryCache}>
+        <ThemeContextProvider>
+          <JotaiProvider>
+            <Layout>
+              <ErrorBoundary>
```

Comment:
Do you think it make more sense to move the ErrorBoundary up so it covers errors in the other ContextProviders?

Reply:
I put it inside the layout so it would be a little styled. I can add another at the outer layer of this to catch any errors with ContextProviders though.

Reply:
👍